packages/cli/main.js

```diff
@@ -15,6 +15,12 @@ async function main() {
       .readAll("utf-8")
   );
 
-  console.log(response.result);
+  if (response.type === "response") {
+    if (response.result !== undefined) {
```

Comment:
Why this test?

Reply:
Because the return of "deleteEntry" or "updateEntry" is undefined. It's useless to post it.

Reply:
I see, unfortunately this is not valid according to the JSON-RPC 2 specification, there should always be result, be it `null` or `true`, even though `json-rpc-protocol` does not enforce this for the moment.

Please fix the server accordingly.